packages/analytics/matomo/matomoPush.ts

```diff
@@ -0,0 +1,14 @@
+declare global {
+   interface Window {
+      _paq?: { push: (args: any[]) => void };
+   }
+}
```

Comment:
Might be better long-term to put this in a `global.d.ts` file, but definitely not a blocker for this. IIRC you can't `declare global` more than once in a package but I could be wrong